docs/pages/build/building-from-github.mdx

```diff
@@ -12,6 +12,23 @@ This guide explains how to trigger builds directly from your GitHub repository u
 
 ## Prerequisites
 
+### Set the `image` field in your eas.json
+
+For the build profiles you want to use with GitHub, you need to specify the image to use for the
+build. This is done in the **eas.json** file in your project's root directory. See [the eas.json reference](/eas/json/#image) to learn more.
+
+```json eas.json
+{
+  // ...
```

Comment:
```suggestion
  /* @hide ... */ /* @end */
```

We use this pattern to tell that there is other parts of this config file for concsistency